runners/direct-java/pom.xml

```diff
@@ -361,6 +361,12 @@
       </exclusions>
       <scope>test</scope>
     </dependency>
-
+    
+    <!-- needed for eclipse-jdt generated core as the test-jar references classes from this -->
```

Comment:
Can you elaborate? DirectRunner shouldn't have any dependencies on Dataflow. /cc @tgroh